app/src/ui/lfs/attribute-mismatch.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react'
+import { Button } from '../lib/button'
+import { ButtonGroup } from '../lib/button-group'
+import { Dialog, DialogContent, DialogFooter } from '../dialog'
+
+interface IAttributeMismatchProps {
+  /** Called when the dialog should be dismissed. */
+  readonly onDismissed: () => void
+
+  /** Called when the user has chosen to replace the update filters. */
+  readonly onUpdateExistingFilters: () => void
+}
+
+export class AttributeMismatch extends React.Component<
+  IAttributeMismatchProps,
+  {}
+> {
+  public render() {
+    return (
+      <Dialog
+        id="lfs-attribute-mismatch"
+        title={
+          __DARWIN__ ? (
+            'Update Existing Git LFS Filters?'
+          ) : (
+            'Update existing Git LFS filters?'
+          )
+        }
+        onDismissed={this.props.onDismissed}
+        onSubmit={this.props.onUpdateExistingFilters}
+      >
+        <DialogContent>
+          <p>
+            Git LFS filters are already configured in your global git config but
```

Comment:
Could we make _global git config_ open the file when clicked? That way people don't have to leave the app to check it out.